tune_sklearn/tune_gridsearch.py

```diff
@@ -192,6 +199,8 @@ def _tune_run(self, config, resources_per_trial):
                 `tune.run`.
 
         """
+        trainable = _Trainable if not self.base_estimator_name else _PipelineTrainable
```

Comment:
(following the above comment)

can we have a method like `check_partial_fit(self.base_estimator)` but:

`check_pipeline(self.base_estimator)`

and this line will become:

```
trainable = _Trainable 
if check_pipeline(self.base_estimator) and self. early_stopping:
    trainable = _PipelineTrainable
```
that way, `TuneSearchCV` won't need to keep track of `self.base_estimator_name`

Reply:
Added, thanks